src/ar_node_worker.erl

```diff
@@ -346,7 +346,7 @@ process_new_block(#{ height := Height } = StateIn, NewGS, NewB, unavailable, Pee
 			StateNext = StateIn#{ gossip => NewGS },
 			process_new_block(StateNext, NewGS, NewB, RecallShadow, Peer, HashList);
 		false ->
-			ar:err(failed_to_get_recall_block),
+			ar:info(failed_to_get_recall_block),
```

Comment:
I think, we should report an error here - a failure to find a recall block should not happen under normal circumstances; also, the weave would stop growing on this node.

Reply:
If it will stall, then I agree with reporting an error here, but to me it doesn't look like it will stall. I think it will continue growing when next block is received.

Please correct me if I'm wrong.

Reply:
> I think it will continue growing when next block is received.

Is it because it can do a fork recovery when the next block arrives?

Yeah, we can use a warning then.

Reply:
Btw, before we created `ar:info`, `ar:err` etc, it was a `ar:report`, so `ar:info` is keeping the old behaviour.

Reply:
> > I think it will continue growing when next block is received.
> 
> Is it because it can do a fork recovery when the next block arrives?
> 
> Yeah, we can use a warning then.

I assume so, yes. But I'm not sure. Keeping the old behaviour (i.e. using `ar:info`) is at least not wrong.

Reply:
If miners' nodes can be stuck without them noticing it, it's a good enough reason for changing the old behaviour. But I am pretty confident fork recovery can bring the node back in this situation, so don't stricly mind keeping info here.